frontend/src/global_styles/content/_toast.sass

```diff
@@ -72,6 +72,10 @@ $nm-upload-box-padding: rem-calc(15) rem-calc(25)
   @include icon-mixin-checkmark
   color: $nm-color-success-icon
 
+%nm-icon-notification
+  @include icon-mixin-bell
+  color: $nm-pb-background-color
```

Comment:
There is a semantic mismatch here. It is kind of a bad style to use the variable for the progress toast for the notification toast as we cannot assure that these will always be of the same color. However, regarding my other ocmment, I think we can remove this whole block altogether.